ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerwidget.js

```diff
@@ -989,12 +989,12 @@ if (typeof window.DatePickerWidget === 'undefined') {
     }
 
     #clearDate(view) {
-      this.setValue("");
-      let existingSelectedItem = this['$'
-      + view.toLowerCase()].getElementsByClassName("dp-selected")[0];
+      this.#model.value = "";
+      let existingSelectedItem = this['$' + view.toLowerCase()].getElementsByClassName("dp-selected")[0];
       if (existingSelectedItem) {
         existingSelectedItem.classList.remove("dp-selected");
       }
+      this.#dp.style.display = "none";
```

Comment:
this is not in parity with AF1, we don't hide date picker dropdown on clicking clear button. But same is happening in HTML5 date picker. 

@rismehta @sakshi-arora1 what should be the expected behaviour.

Reply:
Are we closing the data picker popup on clicking clear button - if so - we should align with AF1.

Reply:
Ok.